pkg/operation/cloudbotanist/alicloudbotanist/controlplane.go

```diff
@@ -129,6 +129,22 @@ func (b *AlicloudBotanist) GenerateKubeSchedulerConfig() (map[string]interface{}
 	return nil, nil
 }
 
+// GenerateETCDStorageClassConfig generates values which are required to create etcd volume storageclass properly.
+func (b *AlicloudBotanist) GenerateETCDStorageClassConfig() map[string]interface{} {
+	return map[string]interface{}{
+		"name":        "etcd-fast",
```

Comment:
Can we find better names for the storage classes that do not contain "etcd"? I guess they can also be used for other purposes

Reply:
Should i simply  keep it  "fast" then.

Reply:
Maybe, but the risk for name conflicts is high then. Add some kind of prefix like` gardener.cloud:fast` (if `:` is allowed)

Reply:
Not allowed. https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
I'll go with "gardener.cloud-fast".